read/parser.go

```diff
@@ -536,34 +522,23 @@ func rawRead(filename string) *Dump {
 	if err != nil {
 		log.Fatal(err)
 	}
-	if prefix || string(hdr) != "go1.3 heap dump" {
-		log.Fatal("not a go1.3 heap dump file")
+	if prefix || string(hdr) != "go1.7 heap dump" {
```

Comment:
This is not adding 1.7 support, it is removing 1.3 support and adding 1.7 support.  I'd rather add functionality instead of replacing it (even if that means more complicated parsing here).